pretty_printers_tests/tests/rc_arc.rs

```diff
@@ -1,13 +1,16 @@
 // min-version: 1.30.0
+// max-version: 1.39.0
```

Comment:
What's the problem with `Rc`/`Arc` pretty printers on 1.40?
Previously we launched tests with 1.40 and they passed

Reply:
They fail on 201 platform with 1.40 for some reason. I'm investigating the problem.

Reply:
Should be fixed now